index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Collaboration Henry & Imran</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    
+      <h1 id="title" style="text-align: center; text-transform: uppercase;">Microverse Survey Form</h1>
+      <p style="text-align: center; text-transform: uppercase; " id="description">This Survey form is intented to collect data of all prospective students of Microverse</p>
+   
+      <div style="margin-top: 4%;"></div>
+
+    <form id="survey-form" action="">
```

Comment:
I think it will be better if you use SECTION tag instead of using DIV. Because the section tag is not a generic container and its a semantic element describing its meaning to both browser and other coding partner(s).